random install failure

**Describe the bug**
pnpm install fails randomly 1 in 5-10 times when using version 5.2.8. Does not happen with 5.2.4

Output:
.../zeromq@5.2.8/node_modules/zeromq install: Building libzmq for linux
.../zeromq@5.2.8/node_modules/zeromq install: build_libzmq.sh: Building version 4.2.2 for architecture x64
.../zeromq@5.2.8/node_modules/zeromq install: gzip: stdin: unexpected end of file
.../zeromq@5.2.8/node_modules/zeromq install: tar: Unexpected EOF in archive
.../zeromq@5.2.8/node_modules/zeromq install: tar: Unexpected EOF in archive
.../zeromq@5.2.8/node_modules/zeromq install: tar: Error is not recoverable: exiting now
.../zeromq@5.2.8/node_modules/zeromq install:   TOUCH Release/obj.target/libzmq.stamp
.../zeromq@5.2.8/node_modules/zeromq install:   CXX(target) Release/obj.target/zmq/binding.o
.../zeromq@5.2.8/node_modules/zeromq install: ../binding.cc:28:10: fatal error: zmq.h: No such file or directory
.../zeromq@5.2.8/node_modules/zeromq install:  #include <zmq.h>
.../zeromq@5.2.8/node_modules/zeromq install:           ^~~~~~~
.../zeromq@5.2.8/node_modules/zeromq install: compilation terminated.

**Reproducing**
Seen with

    pnpm install


**Expected behavior**
Work every time.

**Tested on**
 - OS: Ubuntu 20.04
- pnpm 5.8.0
- zeromq 5.2.8 (OK with 5.2.4)

--- a/scripts/download.js
+++ b/scripts/download.js
@@ -3,8 +3,9 @@ var fs = require("fs");
 var url = require("url");
 
 function writeToFile(filename, response, callback) {
-  response.pipe(fs.createWriteStream(filename));
-  response.on("end", callback);
+  var writeStream = fs.createWriteStream(filename)
+  response.pipe(writeStream);
+  writeStream.on("finish", callback);
 }
 
 function download(fileUrl, filename, callback) {

--- a/scripts/download.js
+++ b/scripts/download.js
@@ -3,7 +3,7 @@ var fs = require("fs");
 var url = require("url");
 
 function writeToFile(filename, response, callback) {
-  var writeStream = fs.createWriteStream(filename)
+  var writeStream = fs.createWriteStream(filename);
   response.pipe(writeStream);
   writeStream.on("finish", callback);
 }

